client/app/controllers/packages/edit.js

```diff
@@ -4,13 +4,11 @@ import { action } from '@ember/object';
 export default class PackagesEditController extends Controller {
   @action
   savePackage() {
-    this.model.saveDescendants();
+    return this.model.saveDescendants();
   }
 
   @action
   submitPackage() {
-    this.model.submit();
-
-    this.transitionToRoute('packages.show', this.model.id);
+    return this.model.submit();
```

Comment:
These probably just need to be async/awaited so the tasks see them.

Reply:
Ooo!